src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -179,8 +188,15 @@ class LocalBackend extends Backend with SharedFileSystem with LazyLogging {
         case Success(0) => processSuccess()
         case Success(143) => AbortedExecution // Special case to check for SIGTERM exit code - implying abort
         case Success(badRc) if !backendCall.call.failOnRc => processSuccess()
-        case Success(badRc) if backendCall.call.failOnRc => FailedExecution(new Throwable(s"Call ${backendCall.call.fullyQualifiedName}, Workflow ${backendCall.workflowDescriptor.id}: $rc\n\nFull command was: ${argv.mkString(" ")}"), Option(badRc))
-        case Failure(e) => FailedExecution(new Throwable(s"Call ${backendCall.call.fullyQualifiedName}, Workflow ${backendCall.workflowDescriptor.id}: $rc\n\nFull command was: ${argv.mkString(" ")}", e))
+        case Success(badRc) if backendCall.call.failOnRc => FailedExecution(
+          new Throwable(s"Call ${backendCall.call.fullyQualifiedName}, " +
+            s"Workflow ${backendCall.workflowDescriptor.id}: $rc\n\n" +
+            s"Full command was: ${argv.mkString(" ")}"), Option(badRc))
+        case Failure(e) =>
+          FailedExecution(new Throwable(s"Call ${backendCall.call.fullyQualifiedName}, " +
+            s"Workflow ${backendCall.workflowDescriptor.id}: $rc\n\n" +
+            s"Full command was: ${argv.mkString(" ")}\n\n" +
+            s"${stderrTailed.tailString}", e))
```

Comment:
I don't disagree with these changes, though I made similar changes (+ more) on my "encrypted workflow options" PR.  Might be easier to just revert this to the way it was so we can use mine.